### PR TITLE
feature/ppt-501: ConvertedPlasticPackagingCredit page

### PR DESCRIPTION
**Pull request by tombart**
- impl for ConvertedPlasticPackagingCredit page
- refactor to follow same pattern for optionals
Screenshot 2021-03-26 at 09 26 17

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -46,7 +46,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 # Custom error handler
 play.http.errorHandler = "uk.gov.hmrc.plasticpackagingtax.returns.config.ErrorHandler"
 
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com www.googletagmanager.com tagmanager.google.com data:"
```
Comment (PauloPortugal) [inline conf/application.conf:49]: This is already in master. Some weird rebase issues:
https://github.com/hmrc/plastic-packaging-tax-returns-frontend/blob/master/conf/application.conf#L49

**Review by PauloPortugal — APPROVED** [review]
Great work.
I would have split the work in two commits: 
 * one to refactor the model format
 * another to deliver the Converted Plastic Packaging Credit work.
 
 Just provides more clarity.

Great stuff :)

File: test/uk/gov/hmrc/plasticpackagingtax/returns/forms/ConvertedPackagingCreditSpec.scala [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+      "totalInPence is withing allowed range" in {
+
+        val input = Map(totalInPence -> "100")
+
+        val form = ConvertedPackagingCredit.form().bind(input)
+        form.errors.size mustBe 0
+      }
+
+      "totalInPence allows decimal values" in {
```
Comment (tmc-ee) [inline test/uk/gov/hmrc/plasticpackagingtax/returns/forms/ConvertedPackagingCreditSpec.scala:44]: Do we need to check that we allow two decimal places only? e.g. `100.99` is allowed, `100.999` is not allowed.

**Review by tmc-ee — APPROVED** [review]

File: conf/application.conf (hunk 1 above)
Comment (tombart) [reply conf/application.conf:49]: yeah not sure why, shouldn't loose anything though.

File: test/uk/gov/hmrc/plasticpackagingtax/returns/forms/ConvertedPackagingCreditSpec.scala (hunk 2 above)
Comment (tombart) [reply test/uk/gov/hmrc/plasticpackagingtax/returns/forms/ConvertedPackagingCreditSpec.scala:44]: yeap agree, added test.

**Pull request merged**